command/rootchain/helper/txn.go

```diff
@@ -48,166 +44,33 @@ func GetRootchainAdminKey() ethgo.Key {
 	return rootchainAdminKey
 }
 
-// SendTxn function sends transaction to the rootchain
-// blocks until receipt hash is returned
-func SendTxn(nonce uint64, txn *ethgo.Transaction, key ethgo.Key) (*ethgo.Receipt, error) {
-	provider, err := getJSONRPCClient()
-	if err != nil {
-		return nil, err
-	}
-
-	txn.GasPrice = defaultGasPrice
-	txn.Gas = defaultGasLimit
-	txn.Nonce = nonce
-
-	chainID, err := provider.Eth().ChainID()
-	if err != nil {
-		return nil, err
-	}
-
-	signer := wallet.NewEIP155Signer(chainID.Uint64())
-	if txn, err = signer.SignTx(txn, key); err != nil {
-		return nil, err
-	}
-
-	data, err := txn.MarshalRLPTo(nil)
-	if err != nil {
-		return nil, err
-	}
-
-	txnHash, err := provider.Eth().SendRawTransaction(data)
-	if err != nil {
-		return nil, err
-	}
-
-	receipt, err := waitForReceipt(provider.Eth(), txnHash)
-	if err != nil {
-		return nil, err
-	}
-
-	return receipt, nil
-}
-
-// Call function is used to query a smart contract on given 'to' address
-func Call(from, to ethgo.Address, input []byte) (string, error) {
-	provider, err := getJSONRPCClient()
-	if err != nil {
-		return "", err
-	}
-
-	callMsg := &ethgo.CallMsg{
-		From:     from,
-		To:       &to,
-		Data:     input,
-		GasPrice: defaultGasPrice,
-		Gas:      big.NewInt(defaultGasLimit),
-	}
-
-	return provider.Eth().Call(callMsg, ethgo.Pending)
-}
-
-func ExistsCode(addr types.Address) (bool, error) {
-	provider, err := getJSONRPCClient()
-	if err != nil {
-		return false, err
-	}
-
-	code, err := provider.Eth().GetCode(ethgo.HexToAddress(addr.String()), ethgo.Latest)
+func ContractExists(client *jsonrpc.Client, contractAddr types.Address) (bool, error) {
+	code, err := client.Eth().GetCode(ethgo.Address(contractAddr), ethgo.Latest)
 	if err != nil {
 		return false, err
 	}
 
 	return code != "0x", nil
 }
 
-func GetPendingNonce(addr types.Address) (uint64, error) {
-	provider, err := getJSONRPCClient()
-	if err != nil {
-		return 0, err
-	}
-
-	nonce, err := provider.Eth().GetNonce(ethgo.HexToAddress(addr.String()), ethgo.Pending)
+// FundAccount funds provided account.
+// Used only for testing purposes.
+func FundAccount(ipAddress string, address types.Address) (types.Hash, error) {
```

Comment:
Same as with `ContractExists`, this function is only here to avoid 2 lines of code. I would inline this on the function call as well. Besides, at least in one instance, the relayer was already defined.

Reply:
Fixed [fb9abbe](https://github.com/0xPolygon/polygon-edge/pull/989/commits/fb9abbe8661f3f160cba5dd828b5fc3b14e542bf)